### PR TITLE
chore: DES-389

### PR DESCRIPTION
**Pull request by aaronkaka**
Publish and integrate into the archetype a decoupled module for centralized control of npm scripts for Origami v2.

Also:
Set up production cheap module source maps.
Add naming convention to READMEs.
Add missing conventional-changelog dependency.

### CODE REVIEW

**Comment by aaronkaka** [comment]
FYI, this is what convinced me to go with 'cheap-module-source-map' for the prod source maps:
http://cheng.logdown.com/posts/2016/03/25/679045

**Comment by StommePoes** [comment]
Cheng's blog is depressing. 7 different default ways of showing *the code you actually wrote*, ostensibly so one can debug, and none of them working very well. Let's go with the hotdogs that only contain sawdust and no rat parts.

**Comment by JasonLantz** [comment]
looks good... solid work Aaron....

PS thanks for updating the naming convention  :)

File: README.md [hunk 1]
```diff
@@ -21,15 +21,16 @@ Otherwise, the repo belongs in Pearson BitBucket, for which you should already h
...
+convention: all lowercase, hyphens instead of camel case.**
 
 2. Follow these [directions](https://help.github.com/articles/caching-your-github-password-in-git/#platform-all) to stop
  manually authenticating to GitHub on every network request. This enables the use of automated npm scripts.
 
 3. Perform these steps in your development environment:  
-	1. git clone https://github.com/Pearson-Higher-Ed/component-archetype.git **[name_of_your_new_component]**  
-	2. cd **[name_of_your_new_component]**  
-    3. git remote set-url origin **[url_of_new_component_repository]**  
+	1. git clone https://github.com/Pearson-Higher-Ed/component-archetype.git `[name_of_your_new_component]`
```
Comment (StommePoes) [inline README.md]: was there a reason this version of the name doens't use hyphens, but underscores?

File: README.md [hunk 2]
```diff
@@ -21,15 +21,16 @@ Otherwise, the repo belongs in Pearson BitBucket, for which you should already h
 
 ## Getting Started
 
-1. Create your new repository in github.com/Pearson-Higher-Ed or bitbucket.pearson.com/projects/ORC.
+1. Create your new repository in github.com/Pearson-Higher-Ed or bitbucket.pearson.com/projects/ORC. **Follow the naming
```
Comment (wyseguyonline) [inline README.md]: With the new project, this should be bitbucket.pearson.com/projects/PDA

Comment (aaronkaka) [inline README.md]: good catch

File: README.md (hunk 1 above)
Comment (aaronkaka) [inline README.md]: it was originally meant to look like spaces, but I can see what you're driving at

**Pull request merged**